pkg/controller/master/image/backing_image_controller.go

```diff
@@ -80,6 +104,9 @@ func (h *backingImageHandler) OnChanged(_ string, backingImage *lhv1beta2.Backin
 			toUpdate.Status.Progress = status.Progress
 			toUpdate.Status.Size = backingImage.Status.Size
 			toUpdate.Status.VirtualSize = backingImage.Status.VirtualSize
+			toUpdate.Status.Failed = 0
+			// vmi is ready if there is one ready file in backing image file map
+			break
```

Comment:
I'm a bit confused about this, so backing image works fine even if not all disk files are in BackingImageStateReady state ?

Reply:
Yes, as there is one ready file, we can leverage the LH backingimage.